transmogrifier/sources/xml/ead.py

```diff
@@ -425,6 +374,147 @@ def get_content_type(cls, source_record: Tag) -> list[str] | None:
 
         return content_types or None
 
+    @classmethod
+    def get_contents(cls, source_record: Tag) -> list[str] | None:
+        contents = []
+        collection_description = cls._get_collection_description(source_record)
+        for arrangement_element in collection_description.find_all(
+            "arrangement", recursive=False
+        ):
+            contents.extend(
+                cls.create_list_from_mixed_value(arrangement_element, ["head"])
```

Comment:
Per our topsy turvy function signature discussion... should `skipped_elements` be named here?

Reply:
Ah, thank you for noting! I updated it in the previous PR, but I think those changes were undone when I rebased the branch with main. 🙃 I'll make the updates (again).